src/pages/s/[area].js

```diff
@@ -1,75 +1,71 @@
 import React, { useEffect } from 'react'
-import { useRouter } from 'next/router'
 import { useQuery } from 'react-query'
-import { getCoordinatesByAddress } from '../../services/Geocoding'
-
+import styled from 'styled-components'
+import axios from 'axios'
```

Comment:
FYI: There are some unused imports in the file, this is what it looks like in IntelliJ.

![IntelliJ](http://screenshots.jonas.re/XACIBQxTMt.png)

Reply:
👍